Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -596,6 +609,87 @@ void HTMLModelElement::setCamera(HTMLModelElementCamera camera, DOMPromiseDeferr
 
 // MARK: - Animations support.
 
+#if ENABLE(MODEL_PROCESS)
+void HTMLModelElement::setPlaybackRate(double playbackRate)
+{
+    if (m_playbackRate == playbackRate)
+        return;
+
+    m_playbackRate = playbackRate;
+
+    if (m_modelPlayer)
+        m_modelPlayer->setPlaybackRate(playbackRate, [&](double) { });
+}
```

Comment:
In the Web Animations spec, there is the distinct notion of the `playbackRate` and an `effectivePlaybackRate` where the former returns the value that is in use at the moment it is requested and thus might differ from what was last set via the API, and the latter whatever was last set via the API, even if it it yet to be committed.

Reply:
With the current code changes, the requested playback rate and the effective playback rate are handled within `ModelProcessModelPlayer`. I'm not sure if that logic should be moved to `HTMLModelElement` instead or if each `ModelPlayer` implementation should decide how they want to implement the animation details.

Reply:
@graouts Do you think this is OK for now until there is clearer text in the spec on handling playback rate?

Reply:
Yes, this is by no mean blocking feedback, just something to consider for asynchronous properties such as `playbackRate`.